bin/projects.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# ===========================================================================
+# Copyright (c) 2019 Eclipse Foundation and others.
+# 
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+# 
+# Contributors:
+# Christopher Guindon (Eclipse Foundation)
+# 
+# SPDX-License-Identifier: EPL-2.0
+# ===========================================================================
+
+./node_modules/js-yaml/bin/js-yaml.js -c data/eclipsefdn_projects.yaml | ./node_modules/json-minify/index.js > static/assets/js/eclipsefdn.projects.json 
```

Comment:
I don't think we need a shell script for this. I think we can run these 2 script via package.json